spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java

```diff
@@ -101,6 +103,7 @@ public class CreateChangelogViewProcedure extends BaseProcedure {
         COMPUTE_UPDATES_PARAM,
         REMOVE_CARRYOVERS_PARAM,
         IDENTIFIER_COLUMNS_PARAM,
+        NET_CHANGES,
```

Comment:
Is netChanges incompatible with "remove_carryovers"?

Reply:
Here is a list of options to generate changelog before this PR:
1. Keeping carry-over rows and not computing pre/post images. I double how useful it is since user can directly use the change table by `select * from table.changes`
2. Removing carry-over rows, but still no pre/post images. this is the default behavior.
3. Removing carry-over rows and computing pre/post images. 

In this PR, we added the fourth option

4. Removing carry-over rows, computing pre/post images, and removing intermediate changes across multiple snapshots.

Does that make sense?

Reply:
Option 1 can be served as a workaround as there is no way to pass options in SQL, so we can't configure snapshot ranges.

Russell does bring a valid point that we can't set `compute_updates` or `remove_carryovers` to false if the new flag is true. Let me think.

Reply:
Maybe, I read it wrong. We take into account `net_changes` only when computing updates?

Reply:
I think it is nice to put `net_changes` under `compute_updates`, even though we can do net changes without compute updates. In a combination way, we provide less options to users for better UX. Users won’t be like “Wow, there are so many knobs, which one or combination should I use.”

I’m open to suggestion though. Let me know how useful is net changes without compute updates. We can provide this option we have a strong reason for that.

Reply:
With new commit, it only do net changes with removing carryovers. We can have a followup PR for computing updates.

Reply:
Is remove_carryover would be better as a three-way enum: 'keep', 'remove' // per snapshot, 'remove_net'.  Right now there's 3 knobs so 8 combos, vs 6?  Or did I mis-understand?

Reply:
Here are combinations:
1. no remove_carryovers
2. remove_carryovers, single-snapshot 
3. remove_carryovers, net change/cross-snapshot
4. compute_update with remove_carryovers, single-snapshot
5. compute_update with remove_carryovers, net change/cross-snapshot

1, 2, 4 have been released. 3 is this PR. 5 is TBD

Reply:
sure, let's discuss this offline a bit.

Reply:
Deprecated the `Remove_CarryOvers` option per discussion.